### PR TITLE
Add starter script for single-task classification fine-tuning

### PR DESCRIPTION
**Pull request by alextrott16**
There is currently a blind-spot in the `bert` folder when it comes to fine-tuning. The GLUE script is too complicated to reflect a typical use-case, so I added `classification.py` as a simple starter script to make it easier for someone to fine-tune a BERT model on some classification task.

In addition to the new script, I have modified the README, added configs, and added a test for the new script.

### CODE REVIEW

File: examples/bert/README.md [hunk 1]
```diff
@@ -129,14 +131,24 @@ composer main.py yamls/test/main.yaml model.name=mosaic_bert
...
 
 ```bash
 # Run the fine-tuning script with the test config and HuggingFace BERT
+python classification.py yamls/test/classification.yaml
+
+# Run the fine-tuning script with the test config and Mosaic BERT
+python classification.py yamls/test/classification.yaml model.name=mosaic_bert
+```
+
+Second, very `glue.py` for both models.
```
Comment (samhavens) [inline examples/bert/README.md:144]: should this be "verify"?

**Review by samhavens — COMMENTED** [review]
I think everyone assumes that "classification" means "sequence classification", but if we ever add a token classification example, maybe we'll wish we'd called this "sequence_classification" instead?

**Comment by jacobfulano** [comment]
This is very helpful! Code lgtm and is a pleasure to read

**Review by jacobfulano — APPROVED** [review]
Clean and easy to read.

**Pull request merged**